scripts/examples/evaluate_from_checkpoint_parallel_synthetic.py

```diff
@@ -27,13 +27,13 @@
 
     if comm.rank == 0:
         log.info(
-            "**********************************************************************\n"
-            "* Multi-Node Random Forest Classification of Balanced Synthetic Data *\n"
-            "**********************************************************************\n"
+            "********************** ************************************************\n"
+            "* Distributed Random Forest Classification of Balanced Synthetic Data *\n"
+            "*********************** ***********************************************\n"
```

Comment:
Are the spaces in the border intentional?

```suggestion
            "***********************************************************************\n"
            "* Distributed Random Forest Classification of Balanced Synthetic Data *\n"
            "***********************************************************************\n"
```